src/Tizen.NUI/src/internal/WebView/WebHitTest.cs

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// It is a class for hit test of web view.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class WebHitTest : Disposable
```

Comment:
In my opinion, it seems that the class is holding data of hit test "result".
How about "WebHitTestResult"?
I think the name "XXXTest" means doing some test so it should provide test methods.

And `HitTestMode` is not used in this class so it'd be better to move to WebView class.

Reply:
Done.